contracts/src/chainlink/cairo/ocr2/aggregator.cairo

```diff
@@ -165,8 +165,8 @@ func constructor{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_p
     let range : Range = (min=min_answer, max=max_answer)
     Aggregator_answer_range.write(range)
 
-    with_attr error_message("decimals exceed 2^8"):
-        assert_lt(decimals, UINT8_MAX)
+    with_attr error_message("decimals are negative or exceed 2^8"):
+        assert_nn_lt(decimals, UINT8_MAX)
```

Comment:
This constant is defined as 255: https://github.com/OpenZeppelin/cairo-contracts/blob/b764660da224ee757b831ea62bf7a4c63ee898fb/src/openzeppelin/utils/constants/library.cairo#L10

So we can just use the standard lib `assert_nn_le` instead of our aux `assert_nn_lt`.

These two are the same:
- `assert_nn_le(decimals, UINT8_MAX)`
- `assert_nn_lt(decimals, 2ˆ8)`

cc @archseer

Reply:
@krebernisak `assert_nn_lt `also check if decimals is not equal to 2^8, `which assert_nn_le`does not. It is ok if decimals == 2^8 ?